packages/@okta/vuepress-site/guides/protect-your-api/configure-packages/net/independ.md

```diff
@@ -1 +1,13 @@
-Install Command or Instructions
\ No newline at end of file
+All of our .NET SDKs are hosted on [NuGet](https://www.nuget.org/). Install the following dependencies in your project:
+
+Install the [Okta.AspNet](https://www.nuget.org/packages/Okta.AspNet) dependency in your project.
```

Comment:
@laura-rodriguez Can you take a look at how I changed this file (removed the list, moved the name of the package into the sentence) and apply it to aspnetcore as well?